src/config/babelrc.js

```diff
@@ -49,6 +49,12 @@ module.exports = () => ({
         {pragma: isPreact ? 'React.h' : undefined},
       ],
     ),
+    ifAnyDep(
+      ['flow-bin'],
+      [
+        require.resolve('@babel/preset-flow'),
```

Comment:
We should add `@babel/preset-flow` to the `dependencies` so folks don't have to install it.